src/ecs/commands/runCommandInContainer.ts

```diff
@@ -26,6 +26,13 @@ export async function runCommandInContainer(
     outputChannel = globals.outputChannel,
     settings: SettingsConfiguration = new DefaultSettingsConfiguration(extensionSettingsPrefix)
 ): Promise<void> {
+    if (!(node instanceof EcsContainerNode)) {
```

Comment:
what is `node` in this case? you can try printing it with `getLogger().error('%O', node)`

What codepath is calling `runCommandInContainer` with something other than `EcsContainerNode` ?